includes/admin.form.inc

```diff
@@ -39,6 +39,13 @@ function islandora_basic_collection_admin(array $form, array &$form_state) {
           '#default_value' => variable_get('islandora_basic_collection_page_size', '12'),
           '#description' => t('The default number of objects to show in a collection view.'),
         ),
+        'islandora_basic_collection_admin_page_size' => array(
+          '#type' => 'textfield',
+          '#title' => t('Default collection objects per page in the admin view'),
+          '#default_value' => variable_get('islandora_basic_collection_admin_page_size', '10'),
+          '#description' => t('The default number of objects to show in the migrate/share/delete interface.'),
```

Comment:
@willtp87 are you referring to the use of "Default" in the #title and #description, or the #default_value?